indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskRunner.java

```diff
@@ -1637,30 +1631,23 @@ public Response setEndOffsets(
           resume();
           return Response.ok(sequenceNumbers).build();
         } else if (latestSequence.isCheckpointed()) {
-          return Response.status(Response.Status.BAD_REQUEST)
-                         .entity(StringUtils.format(
+          return ResponseStatusExceptionMapper.toResponse(Response.Status.BAD_REQUEST,
```

Comment:
_Nit: style choice_
Maybe break the line after `.toResponse(`, would look more readable if the arguments start from a new line.